test/viewmodels/SummaryCardHelperSpec.scala

```diff
@@ -364,70 +361,70 @@ class SummaryCardHelperSpec extends SpecBase with ImplicitDateFormatter {
             sampleRemovedPenaltyPoint
           )
           val expectedResult: Seq[LateSubmissionPenaltySummaryCard] = Seq(LateSubmissionPenaltySummaryCard(
             Seq(
               helper.summaryListRow(
                 period,
-                Html(vatPeriodValue(dateToString(sampleDate), dateToString(sampleDate)))
+                Html(vatPeriodValue(dateToString(sampleDateLSP), dateToString(sampleDateLSP)))
               ),
-              helper.summaryListRow(returnDue, Html(dateToString(sampleDate))),
-              helper.summaryListRow(returnSubmitted, Html(dateToString(sampleDate))),
+              helper.summaryListRow(returnDue, Html(dateToString(sampleDateLSP))),
+              helper.summaryListRow(returnSubmitted, Html(dateToString(sampleDateLSP))),
               helper.summaryListRow(pointExpiration, Html(dateTimeToMonthYearString(LocalDateTime.now)))
             ),
             Tag(content = Text("active")),
             "3",
             "12345678901234",
             isReturnSubmitted = true,
-            dueDate = Some(dateToString(sampleDate))
+            dueDate = Some(dateToString(sampleDateLSP))
           ),
             LateSubmissionPenaltySummaryCard(
               Seq(
                 helper.summaryListRow(
                   period,
-                  Html(vatPeriodValue(dateToString(sampleDate), dateToString(sampleDate)))
+                  Html(vatPeriodValue(dateToString(sampleDateLSP), dateToString(sampleDateLSP)))
                 ),
-                helper.summaryListRow(returnDue, Html(dateToString(sampleDate))),
-                helper.summaryListRow(returnSubmitted, Html(dateToString(sampleDate))),
+                helper.summaryListRow(returnDue, Html(dateToString(sampleDateLSP))),
+                helper.summaryListRow(returnSubmitted, Html(dateToString(sampleDateLSP))),
                 helper.summaryListRow(pointExpiration, Html(dateTimeToMonthYearString(LocalDateTime.now)))
               ),
               Tag(content = Text("active")),
               "2",
               "12345678901234",
               isReturnSubmitted = true,
-              dueDate = Some(dateToString(sampleDate))
+              dueDate = Some(dateToString(sampleDateLSP))
             ),
             LateSubmissionPenaltySummaryCard(
               Seq(
                 helper.summaryListRow(
                   period,
-                  Html(vatPeriodValue(dateToString(sampleDate), dateToString(sampleDate)))
+                  Html(vatPeriodValue(dateToString(sampleDateLSP), dateToString(sampleDateLSP)))
                 ),
-                helper.summaryListRow(returnDue, Html(dateToString(sampleDate))),
-                helper.summaryListRow(returnSubmitted, Html(dateToString(sampleDate))),
+                helper.summaryListRow(returnDue, Html(dateToString(sampleDateLSP))),
+                helper.summaryListRow(returnSubmitted, Html(dateToString(sampleDateLSP))),
```

Comment:
Is it possible to use taxPeriodStart, taxPeriodEnd, taxPeriodDue and receiptDate here?